### DISCUSSION TITLE
QM tests for new features

### DISCUSSION THREAD

**Discussion by lachlanbelcher** [nwchemgit/nwchem#697] [Q&A]
Hello, 
I'm adding some new functionality to NWChem.  I want to be able to test it as part of the qm tests, using runtests.mpi.unix; however, execution of the test will depend not only on the .nw file, but also another density file.  It appears that runtests.mpi.unix does not automatically move this file to the scratch dir to run, and thus my test will fail.  Is there a standard way to tell runtests.mpi.unix that it needs to haul over additional files?  

Thanks

**Comment by hjjvandam** [comment]
The MD test cases need additional files as well (in particular the restart file). You can take a look at how that is done in runtests.mpi.unix. There is no general way I know of to tell this script which additional files to copy over to the run directory.

**Reply by lachlanbelcher** [reply]
Thanks, I'll take a look there.
